submit/samples/sample_checklist/sample_checklist_errors.md

```diff
@@ -0,0 +1,24 @@
+| Error at {<sample-name>}. The value "{<value-provided>}" for field "{<attribute-name>}" does not match regular expression "{<pattern>}"                                          | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must match pattern {<pattern>}                                                                                             |
+| Missing mandatory field "{<property-name>}".                                                                                                                                     | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must have required property {<property-name>}                                                                              |
+| Missing recommended field "{<property-name>}".                                                                                                                                   | **DEPRECATED**. Recommended field validation will no longer be supported.                                                                                                                  |
+| Error at {<sample-name>}. Field "{<property-name>}" may only occur once.                                                                                                         | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must NOT have more than 1 items                                                                                            |
+| Error at {<sample-name>}. Invalid date "{<value-of-property>}" for field "{attribute-name}".                                                                                     | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must match format "date" OR sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must match pattern {<pattern>} |
+| Error at {<sample-name>}. Invalid choice.  The value "{<value-of-property>}" for field "{<attribute-name>}" must contain one of the following values "{<list-of-allowed-values>}" | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must be equal to one of the allowed values:{list-of-allowed-values}                                                        |
+| A symonym was used for field "{<property-name>}".  Replacing value "{<synonym-property-name>}" with value "{<property-name>}"                                                    | **DEPRECATED**. Synonym validation and processing will no longer be supported.                                                                                                             |
+
+
+Please note: The new sample validation service may catch more errors that do not have an equivalent on the old service.
```

Comment:
Is there an example?

Reply:
I don't have an exhaustive list (As I am not sure of all the possible events that may happen validation-wise), but I do have one in mind: If my memory serves me well, XML validation does not validate synonym fields, whereas we added the error on the `oneOf` to ensure only 1 synonym appears in a document

Reply:
This is concerning me:
 **DEPRECATED**. Synonym validation and processing will no longer be supported.

Yes we agreed that going forwards that for any new versions synonyms will not be supported. However for existing(soon to be legacy) checklists  Synonym validation and processing will still need to be supported.
If this is just the error message for the non-legacy versions, that all is golden.